Testing.Configuration/Properties/AssemblyInfo.cs

```diff
@@ -4,7 +4,7 @@
 [assembly: AssemblyDescription("Toolkit for testing configuration file access and custom configuration sections.")]
 [assembly: AssemblyCompany("Affecto")]
 
-[assembly: AssemblyVersion("1.0.0.0")]
-[assembly: AssemblyFileVersion("1.0.0.0")]
+[assembly: AssemblyVersion("1.0.1")]
+[assembly: AssemblyFileVersion("1.0.1")]
```

Comment:
Don't remove the last version number component from AssemblyVersion or AssemblyFileVersion. They should usually be of the format "1.0.1.0". Only AssemblyInformationalVersion should be of the shorter format, and that's the one where we get the package version number from.